uportal-war/src/main/java/net/sf/ehcache/distribution/jgroups/BootstrapRequest.java

```diff
@@ -0,0 +1,161 @@
+/**
+ *  Copyright 2003-2010 Terracotta, Inc.
```

Comment:
@drewwills <https://mvnrepository.com/artifact/net.sf.ehcache/ehcache-jgroupsreplication/1.7>

Because it appears to be used only for uPortal it could be okay to inline, it would be nice to have it broken out into its own subproject within uPortal.
Similar to the decomposition that has been done in #886 .